activities/TankOp.activity/js/activity.js

```diff
@@ -38,30 +38,19 @@ define(["sugar-web/activity/activity"], function (activity) {
 					app.load();
 					app.renderInto(document.getElementById("board"));
 
-					function zoomin(){
-						var x=document.getElementsByClassName("body");
-						for (var i = 0; i < x.length; i++) {
-							x[i].style.bottom="40px";
-						} 
-					}
-
-					function zoomout(){
-						var x=document.getElementsByClassName("body");
-						for (var i = 0; i < x.length; i++) {
-							x[i].style.bottom="0px";
-						} 
-					}
+					play= new TankOp.Play();
 					// Full screen
 					document.getElementById("fullscreen-button").addEventListener('click', function() {
 						document.getElementById("main-toolbar").style.visibility = "hidden";
 						document.getElementById("unfullscreen-button").style.visibility = "visible";
-						document.getElementById("unfullscreen-button").style.top = "40px";
-						zoomin();
+						if (app.init())
```

Comment:
Not a good idea to call init, it should be called once at the activity initialization.

Reply:
Then how should I check the play mode